client/src/lib/data.ts

```diff
@@ -18,13 +17,11 @@ export function readToken(): string {
   return token;
 }
 
-export async function postSignUp(event: FormEvent<HTMLFormElement>) {
-  const formData = new FormData(event.currentTarget);
-  const userData = Object.fromEntries(formData);
+export async function postSignUp(username, password) {
```

Comment:
It looks like you fixed the `event` parameters before seeing my comments. Good! However, you did it because you removed the `form` tag, so maybe you did the right change here but not for the right reason?